rdm/0013-access-levels.md

```diff
@@ -0,0 +1,173 @@
+- Start Date: 2019-11-27
+- RFC PR: [#13](https://github.com/inveniosoftware/rfcs/pull/13)
+- Extends RFC: [#7](https://github.com/inveniosoftware/rfcs/pull/7)
+- Authors: Guillaume Viger
+
+# Access Levels
+
+## Summary
+
+RFC [#7](https://github.com/inveniosoftware/rfcs/pull/7) covers the overall
+access control strategy and the policy implementation details. In particular,
+it covers low-level permissions such as `can_create`, `can_read`, `can_update`
+and so on. This RFC (#13) complements #7 by outlining access levels (typically) on
+a record. Access levels are user-friendly grouping rich with meaning that
```

Comment:
Minor: 

> Access levels are user-friendly grouping rich

Without a comma afterwards is a bit confusing to me. Had to read it a few times :sweat_smile:

Reply:
Reworded differently. Hope it helps.